src/commands/bundles.js

```diff
@@ -1,3 +1,4 @@
+const console = require('../console');
```

Comment:
not sure about overriding global `console`. now i can't run any simple console.logs with my own values for output. how about `logger`?